python/CMakeLists.txt

```diff
@@ -1 +1,15 @@
+# Python cmake, if testing enabled set pip to install test dependencies in venv
+if (KWIVER_ENABLE_TESTS)
+    if (PIP_COMMAND)
+        # list(SUBLIST PIP_COMMAND 0 1 py_pip_command)
+        # list(SUBLIST PIP_COMMAND 1 -1 pip_args)
+        python_target_add_command( "pip-install-upgrade"
+                                   "${PIP_UPGRADE_COMMAND}"
+                                   "Upgrading pip...")
+        python_target_add_command( "pip"
```

Comment:
Worth changing the name to something like `pip-install-deps`? Just don't want this getting confused with the literal command `pip`

Reply:
Can only possibly be helpful.